spikeinterface/widgets/sortingview/amplitudes.py

```diff
@@ -7,7 +7,7 @@
 
 class AmplitudeTimeseriesPlotter(SortingviewPlotter):
 
-    def do_plot(self, data_plot, **backend_kwargs):
+    def do_plot(self, data_plot, hide_unit_selector=False, **backend_kwargs):
```

Comment:
I think it is a mistake.
All params should be visible in main widget even if this params if only for on backend at the begining.
We should not change this signatute at all.  ```def do_plot(self, data_plot, **backend_kwargs):``` 
Otherwise it will be to complicated.